src/osd/OSD.cc

```diff
@@ -9375,8 +9380,6 @@ const char** OSD::get_tracked_conf_keys() const
     "osd_recovery_delay_start",
     "osd_client_message_size_cap",
     "osd_client_message_cap",
-    "osd_heartbeat_min_size",
-    "osd_heartbeat_interval",
```

Comment:
Some of these are in the list to indicate to users that they can be changed at runtime (because they're just read out of the struct), instead of having that scary message about it not being changeable.